tests/ludwig/utils/test_backward_compatibility.py

```diff
@@ -929,3 +929,21 @@ def test_type_removed_from_defaults_config():
 
     for feature_type in config_obj_2.get("defaults"):
         assert "type" not in config_obj_2["defaults"][feature_type]
+
+
+def test_trainer_effective_batch_size():
```

Comment:
Test doesn't fail without your change. Please make sure any time you add a test to fix a bug, it is increasing the surface area of the test suite, and that the test properly fails without the fix applied.